### PR TITLE
Catch exceptions thrown by turbomind inference thread

### PR DESCRIPTION
**Pull request by lvhan028**

### CODE REVIEW

File: lmdeploy/turbomind/turbomind.py [hunk 1]
```diff
@@ -372,7 +377,12 @@ def _async_forward_thread(self, inputs, que: LifoQueue):
             self.gpu_count)
 
         def _func():
-            output = self.model_inst.forward(inputs, instance_comm)
+            try:
+                output = self.model_inst.forward(inputs, instance_comm)
+            except Exception as e:
+                logger.error(f'Exception happened: {e}')
+                self.que.put((-1, None))
```
Comment (AllentDan) [inline lmdeploy/turbomind/turbomind.py:384]: here we used `que` instead of `self.que`

**Review by AllentDan — APPROVED** [review]
LGTM

**Comment by AllentDan** [comment]
Does try catch inference thread influence benchmark performance? @lvhan028

**Comment by lvhan028** [comment]
> Does try catch inference thread influence benchmark performance? @lvhan028

As I tested, it doesn't affect the performance

**Review by lzhangzz — APPROVED** [review]

**Pull request merged**